sass/_FacetBreadcrumb.scss

```diff
@@ -5,8 +5,7 @@
 }
 
 .coveo-facet-breadcrumb-value,
-.coveo-facet-slider-breadcrumb-value,
-.coveo-facet-breadcrumb-value-list-item {
```

Comment:
For the Facet, .coveo-facet-slider-breadcrumb-value is inside of .coveo-facet-breadcrumb-value-list-item, resulting in a double outline